DOF2DMD/Program.cs

```diff
@@ -1,445 +1,1423 @@
+                                string advbordercolor = query.Get("bordercolor") ?? "0000FF";
+                                string advbordersize = query.Get("bordersize") ?? "0";
+                                string animationIn = query.Get("animationin") ?? "none";
+                                string animationOut = query.Get("animationout") ?? "none";
+                                float advtextduration = float.TryParse(query.Get("duration"), out float aresult) ? aresult : 5.0f;
+                                LogIt($"Advanced Text is now set to: {advtext} with size {advsize} ,color {advcolor} ,font {advfont} ,border color {advbordercolor}, border size {advbordersize}, animation In {animationIn}, animation Out {animationOut} with a duration of {advtextduration} seconds");
+                                bool advcleanbg;
+                                if (!bool.TryParse(query.Get("cleanbg"), out advcleanbg))
+                                {
+                                    cleanbg = true; // valor predeterminado si la conversión falla
+                                }
+
+                                if (AdvancedDisplay(advtext, advpath, advsize, advcolor, advfont, advbordercolor, advbordersize, advcleanbg, animationIn, animationOut, advtextduration))
+                                {
+                                    sReturn = "OK";
+                                }
+                                else
+                                {
+                                    sReturn = "Error when displaying advanced scene";
+                                }
+                                break;
+                            case "score":
+                                // [url_prefix]/v1/display/score?player=<active player>&score=<score>&cleanbg=<true|false>
+                                gActivePlayer = int.TryParse(query.Get("player"), out int parsedAPlayer) ? parsedAPlayer : gActivePlayer;
+                                gScore[gActivePlayer] = int.Parse(query.Get("score"));
+                                gNbPlayers = 4;
+                                bool sCleanbg;
+                                if (!bool.TryParse(query.Get("cleanbg"), out sCleanbg))
+                                {
+                                    sCleanbg = true; // valor predeterminado si la conversión falla
+                                }
+
+                                if (DisplayScore(gNbPlayers, gActivePlayer, gScore[gActivePlayer], sCleanbg, gCredits))
+                                {
+                                    sReturn = "Ok";
+                                }
+                                else
+                                {
+                                    sReturn = "Error when displaying score board";
+                                }
+
+                                break;
+                            case "scorev2":

```

Comment:
Looks great. It would make sense to use `[url_prefix]/v2/display/score?players=...`. This is why API versions are for. What we can then do is generate (internally of DOF2DMD) a call to the V2 version with some defaults, when we get a V1 call. What do you think?

Reply:
Good idea, let me change it in the code